src/components/Facility/FacilityMapsLink.tsx

```
@@ -0,0 +1,44 @@
+import { SquareArrowOutUpRight } from "lucide-react";
+import { useTranslation } from "react-i18next";
+
+import { isAndroidDevice } from "@/Utils/utils";
+
+const isValidLatitude = (latitude: string) => {
+  if (latitude === "0E-16") return false;
+  const lat = parseFloat(latitude);
+  return !isNaN(lat) && lat >= -90 && lat <= 90;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)